simplq/src/components/pages/Status/QueueStatus.jsx

```diff
@@ -50,7 +50,7 @@ function QueueStatus(props) {
         handleApiErrors(err);
         timeoutId = setTimeout(update, TIMEOUT);
       });
-  }, [tokenId, oldTokenStatus]);
+  }, [tokenId, oldTokenStatus, showNotification]);
```

Comment:
Let's leave this warning... if we add `showNotification` here, it will lead to an infinite loop call to TokenService.get().

In the long term, all these non presentational code should move out of our components. For example, notification logic should be managed at a single place (https://github.com/SimplQ/simplQ-frontend/issues/163) and similarly for network calls. (https://github.com/SimplQ/simplQ-frontend/issues/163)

I'm hoping that this will then fix the warning.

Reply:
Okay, I'll remove it

Reply:
I should have done this, but while you are at it, can you leave a comment here so that we don't accidentally put this back?